tsconfig.json

```
@@ -1,6 +1,9 @@
 {
 	"compilerOptions": {
 		"allowJs": true,
+		"checkJs": true,
```

`checkJs` implies `allowJs`

```suggestion
		"checkJs": true,
```